### DISCUSSION TITLE
Someone selling BlueSCSIs on eBay in Australia

### DISCUSSION THREAD

**Discussion by boodledog** [erichelgeson/BlueSCSI-v1#181] [General]
There's a guy selling a BlueSCSI 1.0b on eBay. Not sure if this counts but I thought it's worth mentioning.
https://www.ebay.com.au/itm/374291499582?hash=item5725830a3e:g:tvUAAOSwInJjND6b&amdata=enc%3AAQAHAAAAoKF%2FxqyLILQ%2BDFvwUqomQyCPS9xFK8oUoN6qIeMqEHcKIYf%2FVocxfgskLniaGSQCg0Ward8x1ItkNRwC0UifT17NXvzGvyV%2BDMMBUzjvaZgTK0L4I0aGMk7WpvEs9LTisBc2c4Mb9q5FzPJpYx6Wo9MyD0jjSjoDYGoiWZ6GOGYPu1Npthz1r36RNttJfHAXJSM1g6VAgT%2BLW3oSNHRjIyI%3D%7Ctkp%3ABk9SR-jJ0eL2YA

**Comment by erichelgeson** [comment]
Ya that's unfortunate people don't respect licenses. Also unfortunate for anyone who's buying them as he's selling 1.0b's which was a short lived version due to a hardware flaw. If you are in Australia please grab one from Bruce or someone on bluescsi.com.
